apps/src/studio/rocketHeightLogic.js

```diff
@@ -53,7 +53,10 @@ RocketHeightLogic.prototype.onTick = function () {
  * @returns {number} Height of rocket after seconds
  */
 RocketHeightLogic.prototype.rocket_height = function (seconds) {
-  return this.resolveCachedBlock_('VALUE')(seconds);
+  const rocketHeight = this.resolveCachedBlock_('VALUE');
+  if (rocketHeight) {
+    return rocketHeight(seconds);
+  }
```

Comment:
It's a little bit scary to me making code changes in a refactor PR, tho based on the code, this change seems righteous. Would it make more sense to have this change in a separate PR?